### PR TITLE
Capture cloud email & id on feedback

### PR DESCRIPTION
**Pull request by ItzNotABug**
## What does this PR do?

(Provide a description of what this PR does.)

## Test Plan

(Write your test plan here. If you changed any code, please provide us with clear instructions on how you verified your changes work.)

## Related PRs and Issues

(If this PR is related to any other PR or resolves any issue or related to any issue link all related PR and issues here.)

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

(Write your answer here.)

### CODE REVIEW

File: src/lib/components/Feedback.svelte [hunk 1]
```diff
@@ -13,6 +14,10 @@
     async function handleSubmit() {
         submitting = true;
         error = undefined;
+
+        const cloudUserId = loggedIn && $user?.$id ? $user.$id : undefined;
+        const cloudUserEmail = loggedIn && $user?.email ? $user.email : undefined;
```
Comment (stnguyen90) [inline src/lib/components/Feedback.svelte:19]: Why not auto-populate the email field?

Comment (ItzNotABug) [reply src/lib/components/Feedback.svelte:19]: Do we need userId in that case then?

Comment (ItzNotABug) [reply src/lib/components/Feedback.svelte]: I've kept the ID for now, lmk if thats fine else we can remove.

File: src/lib/components/Feedback.svelte [hunk 2]
```diff
@@ -23,7 +28,10 @@
                 email,
                 type: feedbackType,
                 route: $page.route.id,
-                comment
+                comment,
+                metaFields: {
+                    cloudUserId
```
Comment (stnguyen90) [inline src/lib/components/Feedback.svelte:33]: It's probably enough to just make this `userId`

Comment (ItzNotABug) [reply src/lib/components/Feedback.svelte]: Done.

**Review by ArmanNik — APPROVED** [review]

**Pull request merged**